discoverable-client/src/main/resources/application.yml

```diff
@@ -163,3 +163,21 @@ logging:
 
 logbackServiceName: ZWEADC1
 
+---
+spring:
+    profiles: attls
+
+apiml:
+    service:
+        discoveryServiceUrls: http://localhost:10001/eureka/
+        scheme: http
+        customMetadata:
+            apiml:
+                enableUrlEncodedCharacters: true
+                gatewayPort: 10030
+                gatewayAuthEndpoint: /api/v1/gateway/auth
+                corsEnabled: false
```

Comment:
Why do you disable CORS here?

Reply:
this was a copy from original block, it is removed now